erpnext/selling/doctype/customer/customer.py

```diff
@@ -170,7 +170,9 @@ def check_credit_limit(customer, company):
 			throw(_("Please contact to the user who have Sales Master Manager {0} role")
 				.format(" / " + credit_controller if credit_controller else ""))
 
-def get_customer_outstanding(customer, company):
+#PR : 10861, Author : ashish-greycube & jigneshpshah,  Email:mr.ashish.shah@gmail.com 
+# get_customer_outstanding is a very generic function which is invoked from many places. A third non mandatory argument is added to change its behaviour based on caller .
+def get_customer_outstanding(customer, company,caller=None):
```

Comment:
The report name should not be hard-coded like this using caller. Tomorrow, if we need to call this function for another purpose, will not be able to do that. It should be a flag, based on purpose.

Reply:
Always use space after comma